directory/Alex_Okafor_v1.mdx

```diff
@@ -1,35 +0,0 @@
-# Alex Okafor
```

Comment:
Double checking - intentional removal? 👀

Reply:
Yep, I got word from Alex's working partner that they're no longer working as a game developer at the moment. I asked Catt about it [here](https://github.com/QuantumBox/blackgamedevs/issues/220) and got word that it'll be ok to remove them.